manifests/master/windows.pp

```diff
@@ -0,0 +1,24 @@
+# Make sure the windows setup files are on the master for the students
+# to download and configure the Windows Server for the installation of
+# the puppet agent.
+class classroom::master::windows {
+  assert_private('This class should not be called directly')
+
+  File {
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0644',
+  }
+
+  $publicdir   = "/opt/puppetlabs/server/data/packages/public"
```

Comment:
is there a reason why the script needs to live in the same directory as the MSI? for more context, the path you're linking here is owned by the pe_repo class, where the path that we use in the classroom is owned by us.

In any case, all of these config style variables need to come from params.pp